pkg/codegen/internal/test/testdata/external-resource-schema/nodejs/cat.ts

```diff
@@ -0,0 +1,72 @@
+// *** WARNING: this file was generated by test. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+import * as pulumi from "@pulumi/pulumi";
+import { input as inputs, output as outputs } from "./types";
+import * as utilities from "./utilities";
+
+import * as random from "@pulumi/random";
```

Comment:
nit: It doesn't look like the external resource is used here. Not sure if it's worth trying to only import in files where it's used.

Reply:
Yeah, this is actually an existing "issue" -- it happens even with internal references as well. Ideally we wouldn't emit the import if it's not going to be used in the file. Let's fix it, but as a separate change. I'll open an issue.